Formula/gimme-aws-creds.rb

```diff
@@ -0,0 +1,28 @@
+class GimmeAwsCreds < Formula
+  include Language::Python::Virtualenv
+
+  desc "CLI to retrieve AWS credentials from Okta"
+  homepage "https://github.com/Nike-Inc/gimme-aws-creds"
+  url "https://files.pythonhosted.org/packages/2b/7a/e0e905e6f4e7d90b99fcaa94f56a24f67852c75d38399997073b006487cb/gimme%20aws%20creds-2.1.1.tar.gz"
+  sha256 "87ab4a0bd1ce758f24c8302416779c38ca376cf38a58e68fe332512071004c5a"
+
+  depends_on "python"
+
+  def install
+    venv = virtualenv_create(libexec, "python3")
+    system libexec/"bin/pip", "install", "-v", "--no-binary", ":all:",
+                              "--ignore-installed", buildpath
+    system libexec/"bin/pip", "uninstall", "-y", "gimme-aws-creds"
+    venv.pip_install_and_link buildpath
+  end
+
+  test do
+    # Workaround gimme-aws-creds bug which runs action-configure twice when config file is missing.
+    touch ".okta_aws_login_config"
+
+    pipe_output("#{bin}/gimme-aws-creds --action-configure", "TESTPROFILE\nhttps://something.oktapreview.com\n\n\n\n\n\n\n\n\n\n\n")
```

Comment:
Last thing: mind doing an `assert_match "something", pipe_output(...` to verify the output is as expected? Thanks!

Reply:
Added an assert to check that config file is written out. `.okta_aws_login_config`

Reply:
Can you assert on part of the output too? If there's no output or it's not possible for another reason let us know. Thanks!